boards/arm/warp7_m4/doc/warp7_m4.rst

```diff
@@ -0,0 +1,341 @@
+- Audio Codec: NXP SGTL5000 (I2C4 and SAI1 interfaces)
+- S1 - Reset Button (POR_B signal)
+- S2 - User Defined button (ENET1_RD1/GPIO7_IO1 signal)
+- S3 - On/Off (MX7_ONOFF signal)
+- Board to board connector (34 configurable pins)
+- mikroBUS expansion connector
+- 10-pin needle JTAG Connector
+- Debug USB exposing two UARTs (UART1 for A7 and UART2 for M4)
+- MIPI DSI 1 lane Connector
+- LCD Touch Connector (I2C2 interface)
+- Audio Jack: Mic and Stereo Headphone
+
+WaRP7 CPU Board
+===============
+
+- CPU i.MX7 Solo with a Single Cortex A7 (800MHz) core and
+  Single Cortex M4 (200MHz) core
+- Memory
+  - RAM -> A7: 4GB (Kingston 08EMCP04)
+  - RAM -> M4: 3x32KB (TCML, TCMU, OCRAM_S), 1x128KB (OCRAM) and 1x256MB (DDR)
+  - Flash -> A7: 8GB eMMC (Kingston 08EMCP04)
+- Multimedia
+  - MIPI CSI 1 lane connector with 5MP OV5640 camera module (I2C2 interface)
+- Connectivity
+  - Board to board connector (34 configurable pins)
+  - Micro USB 2.0 OTG connector (USB_OTG1 interface)
+  - Murata Type 1DX Wifi IEEE 802.11b/g/n and Bluetooth 4.1 plus EDR
+  (SD1, UART3 SAI2 interfaces)
+- Li-ion/Li-polymer Battery Charger: NXP BC3770 (I2C1 interface)
+- Power management integrated circuit (PMIC): NXP PF3000 (I2C1 interface)
+
+
+For more information about the i.MX7 SoC and WaRP7, see these references:
+
+- `i.MX 7 Series Website`_
+- `i.MX 7 Solo Datasheet`_
+- `i.MX 7 Solo Reference Manual`_
+- `WaRP7 Site`_
+- `WaRP7 Quick Start Guide`_
+- `WaRP7 User Guide`_
+- `WaRP7 GitHub repository`_
+
+Supported Features
+==================
+
+The WaRP7 configuration supports the following hardware features on the
+Cortex M4 Core:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| SENSOR    | off-chip   | fxos8700 polling;                   |
+|           |            | fxos8700 trigger;                   |
+|           |            | fxas21002 polling;                  |
+|           |            | fxas21002 trigger;                  |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
```

Comment:
delete this blank line and remove the spaces before the next line:

    The default configuration can be found in the defconfig file:
    ``boards/arm/warp7_m4/warp7_m4_defconfig``.